src/include/codegen/codegen.h

```diff
@@ -85,6 +85,8 @@ class CodeGen {
   llvm::Constant *NullPtr(llvm::PointerType *type) const {
     return llvm::ConstantPointerNull::get(type);
   }
+  // Wrapper for pointer for constant string
+  llvm::Value *ConstStringPtr(const std::string s) const;
```

Comment:
It doesn't look you have an implementation for this method. Is it used? If not, please remove it.